### ISSUE TITLE
Accessibility: unlabeled element after sort by button

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#915]
**Description**  
When using talkback there is an unlabeled element directly between the sort by button and the post title

**How to Reproduce**  
1. Enable talkback from android settings > accessibility
2. Open thunder amd open a post
3. Single tap the sort by button then swipe right

**Expected Behavior**  
This element should be skipped

**Screenshots**  
![Screenshot_20231123-120839_Thunder](https://github.com/thunder-app/thunder/assets/116814522/67177fd2-fde5-475f-a0c6-de130675a5e2)

**Device & App Version:**  
 - Device: [e.g. Pixel 8, iPhone 13 Pro] oneplus nord n200
 - OS: [e.g. Android 13, iOS 17.2] android 13
 - App Version [e.g. 0.2.5] 0.2.6

**Additional Context**
